### PR TITLE
Update to octoversion config to use main

### PR DESCRIPTION
**Pull request by edlyn-liew-octo**
In Octopus, the the main branch is getting released into pre-release. This PR updates the 'main' branch so that it correctly sends it to the production channel

### CODE REVIEW

File: octoversion.json [hunk 1]
```diff
@@ -1,3 +1,3 @@
 {
-    "NonPreReleaseTags": ["refs/heads/main"]
+    "NonPreReleaseTags": ["main"]
```
Comment (matt-richardson) [inline octoversion.json:2]: Recommend supporting both - thats what we do elsewhere
```suggestion
    "NonPreReleaseTags": ["refs/heads/main", "main"]
```

**Review by matt-richardson — APPROVED** [review]
Noice. :shipit:

File: octoversion.json [hunk 2]
```diff
@@ -1,3 +1,3 @@
 {
-    "NonPreReleaseTags": ["refs/heads/main"]
+    "NonPreReleaseTags": ["refs/heads/main", "main"]
```
Comment (cailyoung) [inline octoversion.json:2]:
```suggestion
    "NonPreReleaseTags": ["main"]
```

We shouldn't need that once we are only building here in Actions.

**Review by cailyoung — APPROVED** [review]
LGTM, one note.

File: octoversion.json (hunk 2 above)
Comment (matt-richardson) [reply octoversion.json:2]: Lols. I just asked Edlyn to add it :)

**Pull request merged**